src/frontend/src/common/redux/types.ts

```diff
@@ -11,10 +14,20 @@ export enum Pathogen {
 export enum CZGEReduxActions {
   SET_GROUP_ACTION_TYPE = "group/setGroup",
   SET_PATHOGEN_ACTION_TYPE = "pathogen/setPathogen",
+  ADD_NOTIFICATION_ACTION_TYPE = "notifications/addNotification",
+  DELETE_NOTIFICATION_ACTION_TYPE = "notifications/deleteNotification",
 }
 
 // persisted names are for use in localstorage
 export enum ReduxPersistenceTokens {
   GROUP = "currentGroup",
   PATHOGEN = "currentPathogen",
 }
+
+export type ReduxNotification = ExposedNotificationProps & {
+  reduxId: number;
```

Comment:
I'm not sure if this is the best place to document it, but it feels like somewhere should mention that the typical use case is `Date.now()` because we want these to be unique. But could also put that comment into `addNotification` instead, especially if it gets the boiler plate `Date.now()` call instead of being in the various dispatches in the components.